modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -50,7 +50,8 @@ However, to allow for quicker detection of unresponsive minions, the timeout val
 You can configure the presence ping parameters in [filename]``/etc/rhn/rhn.conf``, however the default values should be sufficient in most cases.
 
 A lower total presence ping timeout value will increase the chance of false negatives.
-In some cases, a minion might be marked as non-responding, when it is responding, but did not respond quickly enough.
+In some cases, a minion might be marked as non-responding, when it is responding but did not respond quickly enough.
+Additionally, setting this total presence ping timeout value too low could result in a client hanging at the boot screen.
```

Comment:
Similar to previous one. This is specific for the case of retail terminal only, I would recommend changing it from "client" to "retail terminal client".